### DISCUSSION TITLE
Error: " Unable to create the new repository because there are too many new files in this directory"

### DISCUSSION THREAD

**Discussion by laserbeak** [community/community#32758] [Other Feature Feedback, Questions, & Ideas]
Hello,
I am trying to create a repository in an existing directory with github desktop. I get the message:

> Unable to create the new repository because there are too many new files in this directory
![image](https://user-images.githubusercontent.com/3384800/189636950-fc680761-09e0-49ec-a372-63a40e120142.png)

**Comment by christian405893** [comment]
também tenho o mesmo problema

**Reply by squin** [reply]
I also have the same problem. I suppose if the browser does the job then just use that and dump the desktop app.

**Reply by liu-yuxuan-C137** [reply]
I met the same problem. I donot know what the exactly reson. 
But just ignore it. You may see the files needed have already created, just write the summary and commit. It works for me.

**Comment by Allanfernandess** [comment]
eu tambem estou com este problema, por favor ajudem a gente
